> dotnet pack "C:\projects\conventional\src\Core\Conventional.Tests\Conventional.Tests.csproj" --configuration Release --output "C:\Users\appveyor\AppData\Local\Temp\1\4jhi4ddw5c" --no-build
if ($env:APPVEYOR_BUILD_WORKER_IMAGE -eq "Visual Studio 2019") {
cd src\Core
dotnet test
}
else {
dotnet test --filter TestCategory!=Database
}
MSBUILD : error MSB1003: Specify a project or solution file. The current working directory does not contain a project or solution file.

The tests don't look to be running on "Visual Studio 2022" worker.

From looking at the above, I think it will be because the `cd src\Core` is missing from how it's running the tests.

I'll see if I can fix. :)

--- a/src/Core/global.json
+++ b/src/Core/global.json
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "8.0.401",
-    "rollForward": "latestFeature"
+    "version": "8.0.303",
+    "rollForward": "feature"
   }
 }

